subjects/graphql/README.md

```diff
@@ -0,0 +1,185 @@
+## GraphQL
+
+### Objectives
+
+The objective of this project is to learn graphql query language by creating profiles. It will be provided,\
```

Comment:
graphQL = graph query language, so repeating query language after is a bit wierd, maybe putting it after would sound less repeated:

> The objective of this project is to learn the query language [graphQL](https://graphql.org/) by creating your profile page. It will be provided,\

Reply:
ok noted, will fix it